suite-native/transactions/src/components/transactionDetail/TransactionDetailAmount.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { Box, Card, Divider, Text } from '@suite-native/atoms';
+import { formatNetworkAmount } from '@suite-common/wallet-utils';
+import { WalletAccountTransaction } from '@suite-common/wallet-types';
+
+type TransactionDetailAmountProps = {
+    transaction: WalletAccountTransaction;
+};
+
+const TransactionDetailAmountRow = ({ title, value }: { title: string; value: string }) => (
```

Comment:
Why doesn't this have its own component?

Reply:
Haha. It was too small to move to its own cmp and wanted to edit it while styling so it made sense to have it in same file, but I see your point. I'll separate it.